### ISSUE TITLE
The shortcut syntax for an object constructor errors if given the key `end`.

### ISSUE THREAD

**Issue by zck** [jqlang/jq#1664]
The [Object Construction](https://stedolan.github.io/jq/manual/#ObjectConstruction:{}) part of the documentation mentions that the object construction operator `{user: .user, title: .title}` has a syntactical alternative `{user, title}`. This is really cool! It works pretty simply, most of the time:

```
$ echo '{"start": 1, "end": 2}' | jq '{start}'
{
  "start": 1
}
```

But it seems you can't use the string `end` that way.

```
$ echo '{"start": 1, "end": 2}' | jq '{end}'
jq: error: syntax error, unexpected '}', expecting ':' (Unix shell quoting issues?) at <top-level>, line 1:
{end}    
jq: 1 compile error
```

You have to quote `end`, and then it works:
```
$ echo '{"start": 1, "end": 2}' | jq '{"end"}'
{
  "end": 2
}
```

When I was looking for open issues about this, I found [1619](https://github.com/stedolan/jq/issues/1619), but I'm not sure if that's related, because this isn't about `$`-variables. So I'm posting separately. Thanks!

**Comment by Alanscut** [comment]
hi @zck  
This problem has been solved .
jq version : jq  master branch
```
root@os-001: echo '{"start": 1, "end": 2}' | ./jq '{"end"}'
{
  "end": 2
}
```

**Comment by zck** [comment]
@Alanscut If I'm reading this right, that is the way that worked when I originally posted.

What happens if you run this?

```
echo '{"start": 1, "end": 2}' | jq '{end}'
```

**Comment by nicowilliams** [comment]
With master:

```
$ ./jq -cn '{end}'
{"end":null}
$ 
```

1.6, of course, gives an error.

**Comment by itchyny** [comment]
This is fixed by c72ed135e4f1b5a02a8fb3f6cd46f27513ecab2a and released in [1.6](https://github.com/jqlang/jq/releases/tag/jq-1.6) in Nov 2018, so closing this issue.
